package.json

```diff
@@ -37,6 +37,7 @@
         "eslint": "^2.10.2",
         "istanbul": "^0.4.2",
         "jasmine-node": "^1.14.5",
+        "jasmine-node-helpers": "https://github.com/tests-always-included/jasmine-node-helpers",
```

Comment:
We should wait until the repo is pushed to npm and renamed.  After that, this line should simply be

    "jasmine-test-helpers": "^1.0.0",

Or whatever the current version is.